bears/bears-router.js

```diff
@@ -0,0 +1,79 @@
+const router = require('express').Router();
+
+const Bears = require('./bears-model.js'); 
+
+router.get('/', (req, res) => {
+    Bears.find()
+    .then(bear => {
+        res.status(200).json(bear)
+    })
+    .catch(error => {
+        res.status(500).json(error)
+    })
+})
+
+router.get('/:id', validateUserId, (req, res) => {
+    const id = req.params.id;
+    Bears.findById(id)
+    .then(bear => {
+        res.status(200).json(bear)
+    })
+    .catch(error => {
+        res.status(500).json(error)
+    })
+})
+
+router.post('/', (req, res) => {
+    Bears.insert(req.body)
+    .then( newBear => {
+        res.status(201).json(newBear)
+    })
+    .catch(error => {
+        res.status(500).json(error)
+    })
+})
+
+router.delete('/:id', validateUserId, (req, res) => {
+    const id = req.params.id;
+
+    Bears.remove(id)
+    .then( deleted => {
+        const unit = deleted > 1 ? 'records' : 'record';
+        res.status(200).json({ message: `${deleted} ${unit} deleted.` });
```

Comment:
This would be useful if you wanted to be able to remove multiple bears at one time on the front end, but since we're using params to get the id of a single bear it's not very useful. This would make more sense if the route received an array of bear ids to be deleted